test/fixtures.js

```diff
@@ -1,18 +1,192 @@
+                        "version": 1,
+                        "details": {
+                            "default_addr_type": "msisdn",
+                            "personnel_code": "12345",
+                            "addresses": {
+                                "msisdn": {
+                                    "+25682111": {}
+                                }
+                            }
+                        },
+                        "created_at": "2015-07-10T06:13:29.693272Z",
+                        "updated_at": "2015-07-10T06:13:29.693298Z"
+                    }]
+                }
+            }
+        },
+
+        // get contact 082111 by msisdn (passing personnel_code check st-B))
+        {
+            'request': {
+                'method': 'GET',
+                'params': {
+                    'details__personnel_code': '12345'
+                },
+                'headers': {
+                    'Authorization': ['Token test_key'],
+                    'Content-Type': ['application/json']
+                },
+                'url': 'http://localhost:8001/api/v1/identities/search/',
+            },
+            'response': {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [{
+                        "url": "http://localhost:8001/api/v1/identities/cb245673-aa41-4302-ac47-00000000001/",
+                        "id": "cb245673-aa41-4302-ac47-00000000001",
+                        "version": 1,
+                        "details": {
+                            "default_addr_type": "msisdn",
+                            "addresses": {
+                                "msisdn": {
+                                    "+25682111": {}
+                                }
+                            }
+                        },
+                        "created_at": "2015-07-10T06:13:29.693272Z",
+                        "updated_at": "2015-07-10T06:13:29.693298Z"
+                    }]
+                }
+            }
+        },
+
+        // get contact 082111 by msisdn (failing personnel_code check st-B))
+        {
+            'request': {
+                'method': 'GET',
+                'params': {
+                    'details__personnel_code': 'aaaaa'
+                },
+                'headers': {
+                    'Authorization': ['Token test_key'],
+                    'Content-Type': ['application/json']
+                },
+                'url': 'http://localhost:8001/api/v1/identities/search/',
+            },
+            'response': {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": []
+                }
+            }
+        },
+
+
+    // time-out testing
```

Comment:
This fixture probably won't only be used for time-out testing, so rather drop this comment.